static/app/components/smartSearchBar/index.tsx

```diff
@@ -1450,6 +1465,10 @@ class SmartSearchBar extends Component<Props, State> {
     this.onAutoCompleteFromAst(replaceText, item);
   };
 
+  get showSearchDropdown(): boolean {
+    return this.state.loading || this.state.searchGroups.length > 0;
```

Comment:
This isn't used anywhere it looks like. Could there be a possibility of an edge case where there are no search groups but the dropdown is still being displayed?

Reply:
Honestly, I dont know enough about it to say. I'll just drop this entirely